### ISSUE TITLE
IE11 - Map points not displayed on Visualiser even the added service layer appearing under Legend

### ISSUE THREAD

**Issue by IsabellaPan** [fgpv-vpgf/fgpv-vpgf#1561]
Tested build: http://fgpv.cloudapp.net/demo/v1.5.0-8/prod/samples/index-fgp-en.html?keys=JOSM,EcoAction,CESI_Other,NPRI_CO,Railways,Airports,Barley
Test data: http://maps-cartes.ec.gc.ca/arcgis/rest/services/CESI_FGP_All_Layers/MapServer

- Add above test data service layer
- Select Nature, Water, and Air Emissions sub-layers in step 3 as below screenshot
![image](https://cloud.githubusercontent.com/assets/18723160/21730860/a379b65a-d41f-11e6-9ebc-d1b2148992b6.png)
It seems all three sub-layers for CESI are added under Legend, but map points for these layers are not displaying on the map.
![image](https://cloud.githubusercontent.com/assets/18723160/21730943/0ea750b8-d420-11e6-949b-5d8f389f9171.png)

**Comment by IsabellaPan** [comment]
Another observation is that - for some instances, I did the same step as mentioned above, it seems I added CESI layer but the layer visibility was disabled, there was only one layer Nature appearing under it but no symbologies could be seen after expanding Nature, no map points for Nature as well.
In this case CESI layer visibility button cannot toggle to "Show layer". It is simply disabled.
![image](https://cloud.githubusercontent.com/assets/18723160/21777960/4f9c5db0-d66f-11e6-884a-a74dc40b6120.png)

This happened with Firefox as well.
